pom.xml

```diff
@@ -76,10 +76,10 @@ limitations under the License.
     <hamcrest.version>1.3</hamcrest.version>
     <mockito.version>3.6.28</mockito.version>
     <!-- TODO: check if commons-codec was transitively updated to 1.13 and okhttp was updated to 2.7.5 when upgrading-->
-    <beam.version>2.19.0</beam.version>
```

Comment:
Please move the version upgrades to a separate PR

Reply:
https://github.com/googleapis/java-bigtable-hbase/pull/2775 created for upgrading beam version